<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CommandHelpView</class>
 <widget class="QWidget" name="CommandHelpView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>694</width>
    <height>605</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="autoFillBackground">
   <bool>true</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QSplitter" name="mainSplitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>691</width>
     <height>601</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
   <property name="childrenCollapsible">
    <bool>false</bool>
   </property>
   <widget class="QSplitter" name="listSplitter">
	<property name="sizePolicy">
	 <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
	 <horstretch>0</horstretch>
	 <verstretch>0</verstretch>
	 </sizepolicy>
	</property>
	<property name="orientation">
	 <enum>Qt::Horizontal</enum>
	</property>
	<property name="handleWidth">
	 <number>2</number>
	</property>
	<property name="childrenCollapsible">
	 <bool>false</bool>
	</property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <layout class="QVBoxLayout" name="logLayout">
     <property name="spacing">
      <number>3</number>
     </property>
     <item>
      <widget class="QTextEdit" name="symbolLogEdit">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Ignored" vsizetype="Ignored">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QProgressBar" name="symbolProgress">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>15</height>
        </size>
       </property>
       <property name="value">
        <number>0</number>
       </property>
       <property name="textVisible">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
